src/main/java/com/example/employee/model/Employee.java

```diff
@@ -28,68 +28,68 @@ public class Employee {
-        }
-        public EmployeeBuilder department(String department){
-            this.department = department;
-            return this;
-        }
-        public EmployeeBuilder positions(String positions){
-            this.positions = positions;
-            return this;
-        }
-        public EmployeeBuilder email(String email){
-            this.email = email;
-            return this;
-        }
-
-        public EmployeeBuilder is_active(boolean isActive){
-            this.isActive = isActive;
-            return this;
-        }
-
-        public EmployeeBuilder is_pensions_payer(boolean isPensionsPayer){
-            this.isPensionsPayer = isPensionsPayer;
-            return this;
-        }
-
-        public Employee buildEmployee(){
-            return new Employee(this);
-        }
-    }
+//    public Employee(Employee.EmployeeBuilder employeeBuilder) {
+//        this.id = employeeBuilder.id;
+//        this.firstName = employeeBuilder.firstName;
+//        this.lastName = employeeBuilder.lastName;
+//        this.department = employeeBuilder.department;
+//        this.positions = employeeBuilder.positions;
+//        this.email = employeeBuilder.email;
+//        this.isActive = employeeBuilder.isActive;
+//        this.isPensionsPayer = employeeBuilder.isPensionsPayer;
+//    }
+//
+//    @AllArgsConstructor
+//    @NoArgsConstructor
+//    public static class EmployeeBuilder{
+//        private Long id;
+//        private String firstName;
+//        private String lastName;
+//        private String department;
+//        private String positions;
+//        private String email;
+//
+//        private Boolean isActive;
+//        private Boolean isPensionsPayer;
+//
+//        public EmployeeBuilder id(Long id){
+//            this.id = id;
+//            return this;
+//        }
+//        public EmployeeBuilder firstName(String firstName){
+//            this.firstName = firstName;
+//            return this;
+//        }
+//        public EmployeeBuilder lastName(String lastName){
+//            this.lastName = lastName;
+//            return this;
+//        }
+//        public EmployeeBuilder department(String department){
+//            this.department = department;
+//            return this;
+//        }
+//        public EmployeeBuilder positions(String positions){
+//            this.positions = positions;
+//            return this;
+//        }
+//        public EmployeeBuilder email(String email){
+//            this.email = email;
+//            return this;
+//        }
+//
+//        public EmployeeBuilder isActive(boolean isActive){
+//            this.isActive = isActive;
+//            return this;
+//        }
+//
+//        public EmployeeBuilder isPensionsPayer(boolean isPensionsPayer){
+//            this.isPensionsPayer = isPensionsPayer;
+//            return this;
+//        }
+//
+//        public Employee buildEmployee(){
+//            return new Employee(this);
+//        }
+//    }
```

Comment:
Please, remove commented code if it's not needed anymore
We have git history and if we will really need it we can check it in git